sqlite-std.go

```diff
@@ -20,6 +20,97 @@ type DB struct {
 	tmpFile string
 }
 
+func (db *DB) Collection() (*Collection, error) {
+	var deletedDecks []ID
+	if rows, err := db.Query("SELECT oid FROM graves WHERE type=2"); err != nil {
+		return nil, err
+	} else {
+		for rows.Next() {
+			id := new(ID)
+			if err := rows.Scan(id); err != nil {
+				return nil, err
+			}
+			deletedDecks = append(deletedDecks, *id)
+		}
+	}
+	collection := &Collection{}
+	if err := db.Get(collection, "SELECT * FROM col"); err != nil {
+		return nil, err
+	}
+	for _, deck := range collection.Decks {
+		for _, deleted := range deletedDecks {
+			if deck.ID == deleted {
+				delete(collection.Decks, deck.ID)
+				continue
+			}
+		}
+		conf, ok := collection.DeckConfigs[deck.ConfigID]
+		if !ok {
+			//return nil, fmt.Errorf("Deck %d references non-existent config %d", deck.ID, deck.ConfigID)
```

Comment:
This line got commented out during the code move.  Was this intentional?

Reply:
Uops. Yes, you're right. The problem is that in one of my local Anki profiles I had a situation where there was no dek for that `deck.ConfigID`. I don't know how big a problem that incosistency is, but both the desktop Anki client and Ankidroid ignore this feature.

I tried now and it it's not the case any more. I suppose it's probably something left after I removed a deck and it was probably not cleaned immediately but later, after I run the "check database" desktop client. 

I'm removing it for now, but if it happens again, I'll investigate more.